core/src/main/scala/org/broadinstitute/workbench/hamm/dao/GooglePriceListDAO.scala

```diff
@@ -13,34 +13,33 @@ import org.http4s.client.Client
 class GooglePriceListDAO[F[_]: Sync](httpClient: Client[F], uri: Uri) {
 
   def getGcpPriceList(): F[GooglePriceList] = {
+    val serviceId = "6F81-5844-456A"
+    val key = "[KEY]"
     for {
-      googlePriceList <- httpClient.expect[GooglePriceList](uri)
+      googlePriceList <- httpClient.expect[GooglePriceList](uri + s"/v1/services/$serviceId/skus?key=$key")
```

Comment:
is this api key that's needed for authenticating against google pricing api? if so, it probably should be pulled from vault?